packages/alert/src/index.tsx

```diff
@@ -85,28 +102,42 @@ const Alert: React.FC<AlertProps> = ({
       hasTrim={defaults.hasTrim}
       className={className}
     >
-      {defaults.hasIcon && (
-        <Icon>{getAlertIconForIntent(defaults.intent)({theme})}</Icon>
-      )}
-      <div>
-        {typeof title === 'string' ? (
-          <Title as="h4" color="intense">
-            {title}
-          </Title>
-        ) : (
-          title
-        )}
-        {typeof children === 'string' ? (
-          <Description>{children}</Description>
-        ) : (
-          children
+      <BoxInner
+        appearance={defaults.appearance}
+        hasIcon={defaults.hasIcon}
+        closeOnClick={closeOnClick}
```

Comment:
We need 3 props to fully handle closing:
`hasClose: boolean` to control visibility
`onCloseClick: () => void` to handle closing
`closeOnBodyClick: boolean` to handle closing when user clicks on alert body(it's  a little long prop name but it says what it does)